Client/Frontend/Browser/Tab.swift

```diff
@@ -274,7 +274,7 @@ class Tab: NSObject {
         #endif
     }
 
-    func close() {
+    func closeAndRemovePrivateBrowsingData() {
```

Comment:
[cleanup] Rather than two variants of close function (only one of which is ever called externally), there should be one unambiguous close function.